tests/phpunit/includes/options-trait.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use Brain\Monkey\Functions;
+use WP_Syntex\Polylang\Options\Options;
+use WP_Syntex\Polylang\Options\Registry as Options_Registry;
+
+/**
+ * A trait to create new instances of the options.
+ */
+trait PLL_Options_Trait {
+	/**
+	 * Creates a new instance of the options, resets the values, and returns the instance.
+	 *
+	 * @since 3.7
+	 *
+	 * @return Options
+	 */
+	protected static function create_reset_options(): Options {
+		return self::create_options( false );
+	}
+
+	/**
+	 * Returns a new instance of the options.
+	 *
+	 * @since 3.7
+	 *
+	 * @param array|false $options Initial options. Use `false` to delete previous options.
+	 * @return Options
+	 */
+	protected static function create_options( $options = array() ): Options {
+		if ( ! is_array( $options ) ) {
+			// Reset.
+			delete_option( Options::OPTION_NAME );
+		} elseif ( ! empty( $options ) ) {
+			// Use the given options.
+			$prev_options = get_option( Options::OPTION_NAME );
```

Comment:
Do we really need to take care of stored options in the database here?
Could it do more bad than good (regarding potential side effects between tests)?

Reply:
Removed in [37ee8c9](https://github.com/polylang/polylang/pull/1481/commits/37ee8c9e8a61d2efd62e24dd25c76c47e1f3333c).